test/distributed/org/apache/cassandra/distributed/test/SnapshotsTest.java

```diff
@@ -222,6 +224,71 @@ public void testListSnapshotOfDroppedTable()
         instance.nodetoolResult("listsnapshots").asserts().success().stdoutContains("tag1");
     }
 
+    @Test
+    public void testTTLSnapshotOfDroppedTable()
+    {
+        IInvokableInstance instance = cluster.get(1);
+
+        cluster.schemaChange("CREATE KEYSPACE IF NOT EXISTS default WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 2};");
```

Comment:
why rf 2? dont we have just one node? Also, can no you use that distributed keyspace which is created already?